src/tests/unit/tests/common/components/cards/rule-resources.test.tsx

```diff
@@ -2,21 +2,81 @@
 // Licensed under the MIT License.
 import { RuleResources, RuleResourcesDeps, RuleResourcesProps } from 'common/components/cards/rule-resources';
 import { shallow } from 'enzyme';
+import { cloneDeep } from 'lodash';
 import * as React from 'react';
 
 import { exampleUnifiedRuleResult } from './sample-view-model-data';
 
 describe('RuleResources', () => {
-    it('renders', () => {
-        const rule = exampleUnifiedRuleResult;
+    describe('renders', () => {
+        it('with rule url and guidance links', () => {
+            const rule = cloneDeep(exampleUnifiedRuleResult);
 
-        const props: RuleResourcesProps = {
-            rule,
-            deps: {} as RuleResourcesDeps,
-        };
+            const props: RuleResourcesProps = {
+                rule,
+                deps: {} as RuleResourcesDeps,
+            };
 
-        const wrapper = shallow(<RuleResources {...props} />);
+            const wrapper = shallow(<RuleResources {...props} />);
 
-        expect(wrapper.getElement()).toMatchSnapshot();
+            expect(wrapper.getElement()).toMatchSnapshot();
+        });
+
+        it('no url, only guidance links', () => {
+            const rule = cloneDeep(exampleUnifiedRuleResult);
+            rule.url = null;
+
```

Comment:
why not make this a data driven test since u are duplicating the test method code anyways?

the testCase can have the rule value & guidance value.

Reply:
Updated